plugins/scaffolder/src/components/Router/Router.test.tsx

```diff
@@ -26,26 +30,37 @@ import {
   ScaffolderLayouts,
 } from '@backstage/plugin-scaffolder-react';
 import { TemplateListPage, TemplateWizardPage } from '../../alpha/components';
+import { formFieldsApiRef } from '@backstage/plugin-scaffolder-react/alpha';
 
 jest.mock('../../alpha/components', () => ({
   TemplateWizardPage: jest.fn(() => null),
   TemplateListPage: jest.fn(() => null),
 }));
 
+const renderInApp = (element: React.ReactElement, opts?: TestAppOptions) =>
```

Comment:
Maybe this name (`renderInApp `) can be a bit more informative to highlight what the difference is between `renderInApp` and `renderInTestApp` without the need to go to read the code of it.

Reply:
Yeah, for sure. Let me fix!